holesky/.env.example

```diff
@@ -13,6 +13,9 @@ NODE_NUM_WORKERS=1
 NODE_DISPERSAL_PORT=32005
 NODE_V2_DISPERSAL_PORT=32006
```

Comment:
We should also define `NODE_V2_RETRIEVAL_PORT=32007`

Reply:
`NODE_V2_RETRIEVAL_PORT `  are already defined here.

- https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/holesky/.env.example#L22

- https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/mainnet/.env.example#L22